core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/DatabricksParseHelper.scala

```diff
@@ -108,4 +115,41 @@ object DatabricksParseHelper extends Logging {
         Map.empty
     }
   }
+
+  /**
+   * Maps the Photon operator names to Spark operator names using a mapping JSON file.
+   */
+  private lazy val photonToSparkMapping: Map[String, String] = {
+    val jsonString = UTF8Source.fromResource(PHOTON_OPERATOR_MAPPING_FILE).mkString
+    val json = JsonMethods.parse(jsonString)
+    // Implicitly define JSON formats for deserialization using DefaultFormats
+    implicit val formats: Formats = DefaultFormats
+    // Extract and deserialize the JValue object into a Map[String, String]
+    // TODO: Instead of only extracting the first value, we should consider extracting all values
```

Comment:
so is this implying the ones that have multiple entries in the mapping file aren't actually used? seems like a must fix

Reply:
same question here about selecting the first match, is there a plan to fix later?